runtime/kusama/src/lib.rs

```diff
@@ -354,6 +357,8 @@ parameter_types! {
 
 	// miner configs
 	pub OffchainRepeat: BlockNumber = 5;
+
+	pub const VoterSnapshotPerBlock: u32 = u32::max_value();
```

Comment:
```suggestion
    /// The number of voters to fetch, per block, during the snapshot creation. 
    ///
    /// Whilst `UseNominatorsAndUpdateBagsList` or `UseNominatorsMap` is in use, this can still be a very large value. Once the `BagsList` is in full motion, staking might open its door to many more nominators, and this value should instead be what is a "safe" number (e.g. 22500).
	pub const VoterSnapshotPerBlock: NposCompactSolution24::VoterIndex = u32::max_value();
```

This is the roadmap of this storage, best to explain it in all runtimes.

Reply:
b52fd92e101f487fe8d459da979f44b225e1a6c5